student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom)
+    super(age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please make sure to also pass `name` and `parent_permission` in the initializer and pass in the super method, So users can set these values as well for students.

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end